assets/icons/star.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="22"
+   height="22"
+   viewBox="0 0 22 22"
```

Comment:
The figma is organized into pages with a specific icons page. From my sense of things, it's the canonical list of icons: `git log assets/icons` shows that the majority of icons are pulled from this page, there's one counterexample that was delivered over CZO.
Because we needed a filled-version over an outlined-version, I put the version from the icons page through inkscape and did a conversion there (using stroke-to-path to expand the outline and then keeping only the outer shape). The metadata in the svg of `star.svg` betrays that it was output from inkscape, and an existing `read_receipts.svg` shows that it was also made/altered in inkscape.
I think comparing visually they all appear the same. I don't think the defined size in the svg makes a difference in the icon font (`read_receipts.svg` has a height/width of 512, and `hash_sign.svg` has 1024).

Reply:
> Because we needed a filled-version over an outlined-version, I put the version from the icons page through inkscape and did a conversion there (using stroke-to-path to expand the outline and then keeping only the outer shape).

I see. Please definitely include information like that when sending an icon in a PR — that seems like the kind of operation that @terpimost is likely to want to review, or to do himself.

More generally, if you got a thing from a place but then made some changes to it, please don't just say you got it from there and not mention the changes.

When I tried downloading the same icon from the place you said and got a different result, I spent some time (as you see above) spelling out exactly how I got it, because I figured there was a discrepancy we needed to debug between two different ways of getting an icon from Figma. It would have saved me that time to know that the file in the PR wasn't something one could find in Figma at all.


> I think comparing visually they all appear the same.

I don't understand this part. What looks the same as what? The icons in that directory are all different from each other, naturally, and you've just described how you made this icon different from the one at the page you linked to.


> `git log assets/icons` shows that the majority of icons are pulled from this page, there's one counterexample that was delivered over CZO.

(Not that it's the main point, but this is not true: try adding `--stat` to that command and you'll see why. The majority of the icons all come from the first commit there. It looks like I didn't specify this in the commit message, but those icons came from Zulip web.)

Reply:
Understood about documenting any modifications, I will make sure to do that from now.

    I think comparing visually they all appear the same.

What I meant was they all had the same features, rounded corners and a sort of bulged shape, and so appeared to be the same shape when filled (despite the differences in the svg). I've changed it now to source from Zulip web as well, and noted that https://github.com/zulip/zulip/issues/25903 is the original source of the filled version.